depthai_nodes/node/parsers/rf_detr.py

```
@@ -203,31 +227,75 @@ def run(self):
+            final_mask = None
+            mode = self._SEG_MODE if masks is not None else self._DET_MODE
+
+            if mode == self._SEG_MODE:
+                if self.input_shape is None:
+                    raise ValueError(
+                        "RFDETRParser segmentation mode requires model input shape."
+                    )
+
+                final_mask = np.full(self.input_shape, 255, dtype=np.uint8)
+
+                for i, (mask_logits, bbox) in enumerate(zip(masks, boxes_cxcywh)):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)